core/assets/bundles/bundle_de.properties

```diff
@@ -996,7 +1003,7 @@ block.solar-panel.name = Solarpanel
 block.solar-panel-large.name = Großes Solarpanel
 block.oil-extractor.name = Öl-Extraktor
 block.command-center.name = Kommandozentrum
-block.draug-factory.name = Draug Miner-Drohnenfactory
+block.draug-factory.name = Bergbau-Drohnenfabrik
```

Comment:
Hier wuerde ich Draug Bergbau-Drohnenfabrik schreiben, da Draug ein Eigenname des Entwicklers ist und es in zukunft weitere Miner geben koennte, die sich nur um eigenem Namen unterscheiden

Reply:
Draug Bergbau-Drohnenfabrik finde ich gut.